lib/screens/listOfOrgs/listOfOrgsScreen.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/listOfOrgs/widgets.dart';
+
+class ListOfOrganisations extends StatefulWidget {
```

Comment:
1. Try to name the class compactly
```suggestion
class OrganisationList extends StatefulWidget {
```
2. Make sure the name of a widget ends with what it actually is
```suggestion
class OrganizationListScreen extends StatefulWidget {
```
No need to add screen in the name of the file, but add it in the name of the class